src/main/java/io/seqera/tower/cli/Tower.java

```diff
@@ -65,12 +64,6 @@
     @Option(names = {"-t", "--access-token"}, description = "Tower personal access token (TOWER_ACCESS_TOKEN)", defaultValue = "${TOWER_ACCESS_TOKEN}")
```

Comment:
Not related to this issue, but I think that we need to make this field required.